pcf/particle/aws/route53/hosted_zone.py

```diff
@@ -41,12 +41,12 @@ class HostedZone(AWSResource):
 
     UNIQUE_KEYS = ["aws_resource.Name"]
 
-    def __init__(self, particle_definition):
```

Comment:
can we add this to aws_resource so we dont have to edit every particle

Reply:
def __init__(self, particle_definition, session=None):
think you mean this one from cloudfront.py

it already is in aws_resource, but was never used from what I could tell. I think we need to edit every particle to have the session parameter, otherwise there would be no way to pass it in when creating the particle.